app/assets/javascripts/meal_category.js

```diff
@@ -0,0 +1,19 @@
+function MealCategory(data) {
+  this.$radioButtonForm = data.$radioButtonForm;
+  this.$radionButtons = data.$radionButtons;
+}
+
+MealCategory.prototype.bindEvents = function() {
+  var _this = this;
+  this.$radionButtons.on('change', function() {
+    _this.$radioButtonForm.submit();
+  });
+};
+
+$(document).ready(function() {
+  var data = {
+    $radioButtonForm: $("form[data-category='options']"),
+    $radionButtons: $("input[name='category']")
+  }
+  new MealCategory(data).bindEvents();
+});
```

Comment:
Do we need this js? We need to send Ajax request to Store#Index action. Can't we use remote: true option there.

Reply:
Lets keep this for now.